scripts/bootstrap.sh

```diff
@@ -165,7 +165,7 @@ fetchTool()
     xmlFileAsString=`cat "$vcpkgRootDir/scripts/vcpkgTools.xml"`
     toolRegexStart="<tool name=\"$tool\" os=\"$os\">"
     toolData="$(extractStringBetweenDelimiters "$xmlFileAsString" "$toolRegexStart" "</tool>")"
-    if [ "$toolData" = "" ] || [[ "$toolData" == "<?xml"* ]]; then
+    if [[ "$toolData" = "" ] || [[ "$toolData" == "<?xml"* ]]; then
```

Comment:
```suggestion
    if [[ "$toolData" = "" ]] || [[ "$toolData" == "<?xml"* ]]; then
```
for me this looks like also missing a second ] ?

Reply:
good catch - thanks!